### PR TITLE
Amazon plugins: Wrap at 140 columns

### PR DESCRIPTION
**Pull request by nik9000**
Applies our standard column wrapping to the `discovery-ec2` and
`repository-s3` plugins.

### CODE REVIEW

File: plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Wrapper.java
Comment (nik9000) [inline plugins/repository-s3/src/test/java/org/elasticsearch/repositories/s3/AmazonS3Wrapper.java]: This file and its brother below are huge and we might be able to use mockito for it now but I didn't want to make a big change to how the test was implemented. And I was on an airplane and this is the kind of thing you do when you are tired and bored on an airplane. I'd be happy to be told "don't merge this, see about mockito instead" if anyone things that is a good idea.

**Review by ywelsch — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**
